lib/mocha.js

```diff
@@ -677,6 +677,20 @@ Mocha.prototype.forbidPending = function() {
   return this;
 };
 
+/**
+ * Returns Mocha's version.
+ *
+ * @public
+ * @returns {string} module version as specified by "package.json".
+ * @example
+ *
+ * mocha.version();
+ * // => '5.3.0'
+ */
+Mocha.prototype.version = function() {
+  return require('../package').version;
```

Comment:
huh, I always thought this wanted the extension unless it was a `.js` file.

anyway: what does browserify do with this code?

If I'm being honest, I would rather this be a property, something like:

```js
Object.defineProperty(Mocha.prototype, 'version', {
  value: require('../package').version
});

// or writable, for whatever reason
Mocha.prototype.version = require('./package.version');
```

Reply:
> anyway: what does browserify do with this code?

After new commit, works same on both node and browser. Used a `browserify` transform that _only_ keeps the version from "package.json" (for better security).

> If I'm being honest, I would rather this be a property, something like:

Converted to use `Object.defineProperty`. I like this better too.